Include/internal/pycore_runtime_init.h

```diff
@@ -1244,6 +1244,615 @@ extern "C" {
         }, \
     }, \
 }
+
+static inline void
+_Py_StaticStrings_Intern(void) {
+    PyObject *string;
+    string = &_Py_ID(False);
+    PyUnicode_InternInPlace(&string);
```

Comment:
Multiple lines per entry makes this a little harder to read, but that probably doesn't matter too much for generated code.  It might be a little less efficient to store into a variable but it probably isn't significant and I expect the C compiler optimizes that away.  So there probably isn't anything to change here.

That said, it would be nice if we didn't need an intermediate variable for this.  This would require a friendlier version of `PyUnicode_InternInPlace()`, which is outside the scope of this PR.